### DISCUSSION TITLE
Helm - Implement the possibility to add annotations to the deployment

### DISCUSSION THREAD

**Discussion by almightykid** [strimzi/drain-cleaner#125] [Q&A]
Is there a possibility to include annotations in the deployment.yaml file, enabling monitoring tools such as Datadog to access logs and metrics from the pod?

**Comment by scholzj** [comment]
No, that does not seem to be supported. But you can use the YAML installation files and add any annotations you want.

**Reply by rotem-human** [reply]
But we don't see any metric server that is exposing any metrics from this component.
